JSDemos/Demos/FileManager/UICustomization/jQuery/data.js

```diff
@@ -86,3 +86,21 @@ const fileSystem = [
     size: 2048,
   },
 ];
+
+const fileExtensions = {
+  'Text Document': '.txt',
+  'RTF Document': '.rtf',
+  Spreadsheet: '.xls',
```

Comment:
should work, but lets make 'Spreadsheet' key to be a string explicitly for uniformity